bin/install-dependencies.bash

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Installs dependency for Ubuntu Linux, which is used on Github Actions
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Print functions
+stdmsg() {
+  local IFS=' '
+  printf '%s\n' "$*"
+}
+
+errmsg() {
+  stdmsg "$*" 1>&2
+}
+
+# Trap exit handler
+trap_exit() {
+  # It is critical that the first line capture the exit code. Nothing
+  # else can come before this. The exit code recorded here comes from
+  # the command that caused the script to exit.
+  local exit_status="$?"
+
+  if [[ ${exit_status} -ne 0 ]]; then
+    errmsg 'There is an error installing the dependencies.'
+    exit 1
+  fi
+}
+trap trap_exit EXIT
+
+# Returns 0 if the environment variable is set to any value including
+# null, 1 otherwise.
+is_set() {
+  [[ -n ${!1+x} ]]
+}
+
+# Update package list
+apt-get update
+
+# Install wget and shellcheck
+apt-get update
+apt-get install wget xz-utils -y
```

Comment:
Let me try updating my test environment to a newer version (or should I say, 'less archaic' version 😅), I think it may be due to using an outdated version of Ubuntu for testing resulted in an old version of curl being installed.